backend/inc/class.fileaccess.php

```diff
@@ -356,7 +356,7 @@ function get_parent_directories($iddirectory, $idclient) {
 	//						-> else: only $file is used as ID
 	//
 	function get_file($file, $idclient = '', $iddirectory = '', $type = 1) {
-		if (empty($file)) return ''; // missing fileid
+		if (empty($file)) return array(); // missing fileid
```

Comment:
Why do you use `array()` instead of `[]` in this file (2 times)?

Reply:
I tried to be compatible to php < 5.4. I dont know if this still makes sense.